reactioncommerce/reaction
# refactor: remove inviteShopMember meteor method and rewrite with GraphQL

Resolves #5337   
Impact: **minor**  
Type: **feature|refactor**

## Issue
`inviteShopMember ` still uses a meteor method to perform its actions.

## Solution
Rewrite `inviteShopMember ` mutation to use GraphQL instead of meteor method.
- Remove `inviteShopMember ` meteor method from app

## Breaking changes
Nothing in core code. Any custom code calling the `accounts/inviteShopMember ` Meteor method will break.

## Testing
1. In Reaction Admin, invite a new user (non-registered) to a group
1. See that the invite worked
1. Invite a user with an existing account, but unverified email address to a group
1. See that the invite worked
1. Invite a user with an existing account with a verified email address to a group
1. See that the invite worked
